src/map/lua/luautils.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*
+/*
```

Comment:
theres a lot of files with changes in this line. Is this adding that red dot thing?

Reply:
Byte order mark stuff, I don't remember how we deal with this